macros/parsers/parserCheckboxList.pl

```diff
@@ -447,7 +463,8 @@ sub cmp_defaults {
 		list_type         => 'selection',
 		requireParenMatch => 0,
 		implicitList      => 0,
-		correct_choices   => $self->data
+		correct_choices   => $self->data,
+		correct_values    => $self->{values}
```

Comment:
`$self->{values}` will be all values, not just the correct values.  `$self->data` is actually the `correct_values`.  What exactly are you trying for here?